project/Dependencies.scala

```diff
@@ -2,18 +2,17 @@ import sbt._
 
 object Dependencies {
   val circeVersion = "0.13.0"
-  val http4sVersion = "0.21.7"
-  val grpcCoreVersion = "1.28.0"
-  val scalaTestVersion = "3.2.2"
-  val workbenchGoogle2V = "0.13-39c1b35"
+  val http4sVersion = "0.21.20"
+  val grpcCoreVersion = "1.34.0"
+  val scalaTestVersion = "3.2.7"
+  val workbenchGoogle2V = "0.20-52e271f"
```

Comment:
Latest `google2` will be `0.21-xx` when this is merged btw: https://github.com/broadinstitute/workbench-libs/pull/598